eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/ConnectionParameters.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import com.azure.core.amqp.TransportType;
+import com.azure.eventhubs.implementation.TokenProvider;
+import reactor.core.scheduler.Scheduler;
+
+import java.time.Duration;
+import java.util.Objects;
+
+/**
+ * A wrap class that contains all parameters that need to establish a connection to an event hub.
```

Comment:
nit: A wrapper class that contains all parameters that are needed to establish a connection to an Event Hub.